src/SFA.DAS.FAT.Domain/Courses/Api/Requests/GetCourseApiRequest.cs

```diff
@@ -10,7 +10,7 @@ public class GetCourseApiRequest : IGetApiRequest
         private readonly string _locationName;
         private readonly Guid? _shortlistUserId;
 
-        public GetCourseApiRequest (string baseUrl, int id, double lat, double lon, string locationName, Guid? shortlistUserId = null)
+        public GetCourseApiRequest(string baseUrl, int id, double lat, double lon, string locationName, Guid? shortlistUserId = null)
```

Comment:
update lat to latitude, lon to longitude (but the conversation about using location might supercede this?)